dist/localization.json

```
@@ -357,7 +358,7 @@
     "none": "none",
     "warns_count": "Всего: {0} предупреждений",
     "warn_remove_error_null_or_negative": "ID предупреждения {0} не может быть меньше или равно нулю.",
-    "warn_remove_error_notExistant": "Предупреждение {0} не существует",
+    "warn_remove_error_notExistent": "Предупреждение {0} не существует",
```

Follow one format, change camelCase to snake_case.
```suggestion
    "warn_remove_error_not_existent": "Предупреждение {0} не существует",
```